Travel_fellowships.md

```diff
@@ -1,41 +1,183 @@
+
+### Items covered for remote participation
+
+The fellowship can be used to cover conference registration fees and the costs of small hardware (such as Microphone, speaker and webcam), childcare for the duration of the event and/or other supporting materials.
+
+The cost of remote participation will be evaluated on a case-by-case basis.
+For example, we are aware that due to the load-shedding issues in many developing countries, our applicants might require to pay for the fuel for the generator to keep their electricity running for the duration of the event.
+Similarly, the cost of high-speed internet may vary across countries and may also be requested by applicants from many developing countries.
+
+You are eligible to apply for the cost of a communication platform only if you are hosting a virtual event and will require a paid subscription to access the full feature of the tool.
+When applying for the cost of hardware, please note that this fellowship will not cover the cost of buying laptops and computers.
+However, this fellowship will pay for the rental cost of the large hardware required to attend or host a virtual event.
+The cost will be capped to a maximum of $40 (USD) each for small hardware such as headphones, webcam, speaker, and/microphone.
+You are welcome to negotiate the cost of your hardware with the fellowship chair if the maximum cap is insufficient for you.
+
+If you anticipate that you would require more than USD 1,000 to cover your overall cost, please contact the fellowship chair with the details.
+
+## Review process
+
+### Role of the fellowship chair (and review timeline)
+
+The fellowship chair will make sure that the call for applications is announced at least 6 weeks before the deadline by editing and updating the application form for each round, drafting a blog post, coordinating with the newsletter chair and receiving approval from the OBF board members.
+
+After the application period is over, they will coordinate with the OBF board members to invite volunteers to serve in the review panel and create all the required documents such as applications (minus the personal information) and reviewing form with rubrics to share with the panel.
+In the first week after the deadline, the review panel will be asked to review all or selected applications (based on the number of applications for that round).
+
+In the second week, the fellowship chair will create a summary of the scores and discuss it with the board members to identify the awardees for that round.
+
+Preferably, by the beginning of the third week, the fellowship chair will inform the selected asking for confirmation regarding the fellowship acceptance and informing them of their responsibilities for getting reimbursed for their expense after they have participated in the event of their choice.
```

Comment:
inform the selected asking -> inform the chosen awardees
their responsibilities for -> the process for
the event of their choice -> the event.  (don't want to make it sound like now they can choose a different event!)